themes/custom/droopler_theme/scss/components/_d_p_gallery.scss

```diff
@@ -46,6 +46,39 @@ $d-p-gallery-bold-color: $color-troilus !default;
   }
 
   .field--name-field-d-media-image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .field__item {
+      display: flex;
+      justify-content: center;
+
+      div {
+        width: 100%;
+      }
+    }
+
+    a[href*=".svg"] {
+      position: relative;
+      width: 100%;
+
+      &::before {
+        content: '';
+        display: block;
+        padding-bottom: calc(100% / (320 / 240) + 1em);
```

Comment:
Mógłbyś rzucić nieco światła z czego wynikają te wartości :-)?

Reply:
proporcje obrazka + margin na img
jak chcesz np. obrazek 16:9 to robisz padding-bottom: calc(100% / (16 / 9));

Reply:
No dobra, ale to że my tak mamy w Drooplerze, to nie znaczy, że ktoś sobie nie ustawił zupełnie innego rozmiaru obrazka w stylach, dobrze myślę?

Reply:
jeśli ma to działać dla każdego wymiaru a nie być tak zhardcodowane jak tutaj to pierwsze co mi przychodzi do głowy do varsy cssa, do htmla wrzucamy inlineowo variablesa style="--aspect-ratio:320/240;" i w css
@supports (--custom:property) {
  padding-bottom: calc(100% / (var(--aspect-ratio)));
}

Reply:
^ brak wsparcia dla ie11

Reply:
No mi raczej przychodzi przekazywanie wymiarów do HTMLa jako inlinowe style.

Reply:
Ale nie wiem do końca co tu ma być wykonane i skąd co się bierze, czy ten SVG to nie moze miec jakiegos formattera? Nie da się tego załatwić bez before?

Reply:
W sumie na jednym z projektów robiłem tak formatter do svga, zeby mu robiło odpowiednie wymiarowanie (proporcje). Musiałbym poszukać. W każdym razie da się.